server/src/main/java/io/spine/server/model/MethodParams.java

```diff
@@ -20,91 +20,90 @@
-     * @param type
-     *         the type of expected single parameter
-     * @return {@code true} if the method has exactly one parameter of the expected type,
-     *         {@code false} otherwise
+     * @param index the zero-based index of the type.
      */
-    public static boolean consistsOfSingle(Class<?>[] methodParams, Class<?> type) {
-        checkNotNull(methodParams);
-        checkNotNull(type);
+    public Class<?> type(int index) {
+        Parameter parameter = params.get(index);
+        return parameter.getType();
+    }
 
-        if (1 != methodParams.length) {
+    /**
+     * Verifies if the method has only one parameter of the passed type.
+     */
+    public boolean is(Class<?> type) {
+        if (size() != 1) {
             return false;
         }
-        Class<?> firstParam = methodParams[0];
+        Class<?> firstParam = type(0);
         return type.isAssignableFrom(firstParam);
     }
 
     /**
-     * Checks whether the given method parameters are two parameters of expected types.
-     *
-     * <p>This is a convenience shortcut for {@link #consistsOfTypes(Class[], List)
-     * consistsOfTypes(Class[], List)}.
-     *
-     * @param methodParams
-     *         the method parameters to check
-     * @param firstType
-     *         the expected type of the first parameter
-     * @param secondType
-     *         the expected type of the second parameter
-     * @return {@code true} if the method parameters are of expected types, {@code false} otherwise
+     * Verifies if the method has only two parameters and they match the passed types.
      */
-    public static boolean consistsOfTwo(Class<?>[] methodParams,
-                                        Class<?> firstType,
-                                        Class<?> secondType) {
-        checkNotNull(methodParams);
-        checkNotNull(firstType);
-        checkNotNull(secondType);
-
-        return consistsOfTypes(methodParams, asList(firstType, secondType));
+    public boolean are(Class<?> type1, Class<?> type2) {
+        if (size() != 2) {
+            return false;
+        }
+        boolean firstMatches = type1.isAssignableFrom(type(0));
+        boolean secondMatches = type2.isAssignableFrom(type(1));
+        return firstMatches && secondMatches;
     }
 
     /**
-     * Checks whether the given method parameters are of expected types.
-     *
-     * @param methodParams
-     *         the method params to check
-     * @param expectedTypes
-     *         the expected types
-     * @return {@code true} if the method parameters are of expected types, {@code false} otherwise
+     * Verifies if these parameters match the passed types.
      */
-    public static boolean consistsOfTypes(Class<?>[] methodParams,
-                                          List<Class<?>> expectedTypes) {
-        if (methodParams.length != expectedTypes.size()) {
+    public boolean match(Class<?>... types) {
```

Comment:
Maybe, instead of having this overload, expand the signature of `are`  to 
```java
public boolean are(Class<?> type1, Class<?> type2, Class<?>... rest)
```

Guava uses that trick in many places, e.g. `ImmutableList.of(...)`.